apm-agent-plugin-sdk/pom.xml

```diff
@@ -56,6 +56,14 @@
                     </archive>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
```

Comment:
Since co.elastic.apm.agent.sdk.ElasticApmInstrumentation is in this module, Java 7 is (effectively) no longer supported after this PR. Maybe source and target should be bumped in the parent POM?

Reply:
I think that's a bug. Only the OTel module should be compiled with target 8, the rest should say as-is.